scipy/signal/tests/test_fir_filter_design.py

```diff
@@ -13,26 +15,26 @@
 
 def test_kaiser_beta():
     b = kaiser_beta(58.7)
-    assert_almost_equal(b, 0.1102 * 50.0)
+    assert_almost_equal(np.asarray(b), np.asarray(0.1102 * 50.0))
```

Comment:
if we just need to compare python scalar against python scalar, perhaps `pytest.approx` would be better than `assert_almost_equal` here?

Reply:
Definitely -1 for `pytest.approx`. Why would we want to mix in some other framework with who knows what defaults and quirks. 

This twice np.asarray is a leftover from before having to add these crossed the annoyance threshold, thanks for spotting it. Will remove.

Reply:
> Definitely -1 for pytest.approx. Why would we want to mix in some other framework with who knows what defaults and quirks.

That's fair. To be honest this was prompted by a conversation I had at the summit yesterday about a project which uses `pytest.approx` a lot. But yeah, I think I agree that it is nice for the assertions to be able to handle both arrays and scalars cleanly.